Projects/UOContent/Commands/Handlers.cs

```diff
@@ -21,7 +21,7 @@ public static class CommandHandlers
     {
         public static void Initialize()
         {
-            CommandSystem.Prefix = "[";
+            CommandSystem.Prefix = ServerConfiguration.GetSetting("command.prefix", "[");
```

Comment:
Also a nitpick, but I am wondering if we should make the feature `commandsystem` so the config would be `commandsystem.prefix`.

Thoughts?

Reply:
Sounds good :)